django_project/lesson/models/further_reading.py

```diff
@@ -0,0 +1,37 @@
+# coding=utf-8
+"""Further reading model definitions for lesson apps.
+
+"""
+import logging
+
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from lesson.models.worksheet import Worksheet
+
+logger = logging.getLogger(__name__)
+
+
+class FurtherReading(models.Model):
+    """Further reading lesson model."""
```

Comment:
This is a forward link for the topic so that the lesson reader can find more info about the topic.